__version__.py

```diff
@@ -1 +1 @@
-__version__ = "0.1.1a46"
+__version__ = "0.1.1a47"
```

Comment:
Update this to 0.1.1a48 please as I've forced a publish of 0.1.1a47 to unblock Stephen with the lusidjam work.

Reply:
a49